test/linter/test-browsers-data.js

```diff
@@ -25,8 +24,8 @@ function processData(data, logger) {
 
   for (const status of ['current', 'beta', 'nightly']) {
     const releasesForStatus = Object.entries(releases)
-      .filter(([version, data]) => data.status == status)
-      .map(([version, data]) => version);
+      .filter(([_, data]) => data.status == status)
```

Comment:
Couldn't we just omit `_` here, to avoid the `no-unused-vars` exception in #5904?

```suggestion
      .filter(([, data]) => data.status == status)
```

Reply:
Oh right, I forgot you can do that in JavaScript!